app/src/main/java/com/celzero/bravedns/service/PersistentState.kt

```diff
@@ -100,10 +102,11 @@ class PersistentState(private val context: Context):SimpleKrate(context) {
     var numberOfBlockedRequests by intPref("blocked_request", 0)
     var backgroundEnabled by booleanPref("background_mode", false)
     var checkForAppUpdate by booleanPref("check_for_app_update", true)
-        //private set
     var isScreenOff by booleanPref("screen_off", false)
     private var connectedDNSName by stringPref("connected_dns_name","RethinkDNS Basic")
     var theme by intPref("app_theme", 0)
+    var notificationAction by intPref("notification_action", 1)
+    var isAddAllNetworks by booleanPref("add_all_network_VPN", false)
```

Comment:
nit: rename the preference key to `add_all_networks_to_vpn`...

Reply:
Renamed.